pages/town-hall/f22.js

```diff
@@ -96,26 +81,26 @@ function TownHall() {
         </p>
         <ul className="list-unstyled text-center">
           <li style={inlineButtonListStyle}>
-            <Link href="https://forms.gle/Rsh1ZVBSJnzsYqf68">
+            <Link href="https://docs.google.com/presentation/d/14MIk1bzHHr5b11cgYX_kAz5WQa6ToKD_9nnj3_5DLco/edit?usp=sharing">
               <a className="button" target="_blank" rel="noopener noreferrer">
                 <FontAwesomeIcon
                   icon={faCalendarCheck}
                   fixedWidth
                   aria-hidden={true}
                 />{' '}
-                RSVP now!
+                Form Summaries and Slides
               </a>
             </Link>
           </li>{' '}
           <li style={inlineButtonListStyle}>
-            <Link href="#surveys">
-              <a className="button">
+            <Link href="https://docs.google.com/document/d/1JTFaP27OsqA0sBhjbIC4mIzjCgqRLJyBxvOQ5FSrPCk/edit?usp=sharing">
+              <a className="button" target = "_blank">
```

Comment:
Very small nit, this a tag is missing a `rel="noopener noreferrer`!

Reply:
ah yes, I saw the tag but didn't include it because [of this update](https://chromestatus.com/feature/6140064063029248). Please lmk if it's still good practice to add it and I'll make the change

Reply:
Oh, great! Thanks for letting me know, in that case that's not necessary I did not know about that new update!